src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,39 @@
 package practice;
 
-public class CandidateValidator {
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+    private static final String NATIONALITY = "Ukrainian";
+    private static final String YEARS_DELIMITER = "-";
+    private static final int LOWER_AGE_LIMIT = 35;
+    private static final int MIN_RESIDENCY_YEARS = 10;
+    private static final int FROM_YEAR = 0;
+    private static final int TO_YEAR = 1;
```

Comment:
Well done with extracting to constants, but this isn't a year, it's more of an index of the year :)
```suggestion
    private static final int FROM_YEAR_INDEX = 0;
    private static final int TO_YEAR_INDEX  = 1;
```